.github/workflows/linter.yml

```diff
@@ -22,11 +22,10 @@ jobs:
           fetch-depth: 0
 
       - name: Lint Code Base
-        uses: github/super-linter@v5
+        uses: super-linter/super-linter@v5
         env:
           VALIDATE_ALL_CODEBASE: false
-          DEFAULT_BRANCH: master
```

Comment:
The default values for these env vars are the same as the set values here. See [here](https://github.com/super-linter/super-linter#environment-variables).